sdk/communication/Azure.Communication.MediaComposition/src/Models/ImageInput.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+
+namespace Azure.Communication.MediaComposition.Models
+{
+    /// <summary>Image input.</summary>
+    [CodeGenModel("Image")]
+    public partial class ImageInput
+    {
```

Comment:
Is there a flag we can set to not generate these "empty" files? Are they really useful?

Reply:
This is the way to rename Image -> ImageInput because the .Net SDK does not allow for single word classes as it causes ambiguity